udata/frontend/markdown.py

```diff
@@ -39,20 +42,42 @@ def nofollow_callback(attrs, new=False):
     That callback is not splitted in order to parse the URL only once.
     """
 
+    print(f'\n>>>>> nofollow_callback / attrs : \n{attrs}')
+    # print(f">>>>> nofollow_callback / current_app.config['SERVER_NAME'] : \n{current_app.config['SERVER_NAME']}")
+
     if (None, u"href") not in attrs:
+        print(f'>>>>> nofollow_callback / attrs X : \n{attrs}')
         return attrs
+
     parsed_url = urlparse(attrs[(None, 'href')])
+    print(f'>>>>> nofollow_callback / parsed_url : \n{parsed_url}')
+
+    if parsed_url.scheme == u"mailto" :
```

Comment:
Unicode string are defaults in python3, no need for `u''`

Reply:
ok